Matveibos/hometask-3/rainfall_stat.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# this class is used for calculate mean and variance values for town rainfall
+class RainfallStat
+  SEPARATOR = ':'
+  CHOOSING_NUMBERS = /\d*\.\d/.freeze
+
+  def initialize(town, data)
+    @town = town
+    @data = data
+    @processed_data = {}
+    create_hash
+  end
+
+  def mean
+    calc_mean
```

Comment:
Do we need to have this method, just to call another method? Maybe we can just move `calc_mean` method content here?